main.js

```diff
@@ -0,0 +1,275 @@
+/******CONSTANTS*******/
+const Medium = document.getElementById("Medium");
+const Hard = document.getElementById("Hard");
+const Easy = document.getElementById("Easy");
+const highScore = document.getElementById("high-score");
+const Score = document.getElementById("score");
+const speedRadios = document.querySelectorAll('input[name="speed"]');
```

Comment:
In here I see too much incoherence:
- Stick to one rule for choosing IDs, in your case, it is snake case separated by a "-", so IDs shall be: "medium, hard, easy, high-score, score" without any uppercase letter.
- There is no convention within Javascript where develops name their variables starting with an uppercase followed by lowercase letters. In this case your variables shall be: "mediumButton, hardButton, easyButton, highScoreText, scoreText, speedRadio". See here, I gave your variables meanings where each item is followed by the variable time as "medium" for example didn't give any meaning as for what the variable is holding.